src/test/java/cz/muni/fi/pa165/library/repositories/BookRepositoryTest.java

```diff
@@ -1,42 +1,96 @@
 package cz.muni.fi.pa165.library.repositories;
 
 import cz.muni.fi.pa165.library.entities.Book;
-import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
-import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.dao.InvalidDataAccessApiUsageException;
 import org.springframework.test.context.junit4.SpringRunner;
-
+import java.util.Arrays;
 import java.util.List;
 
-import static org.hamcrest.MatcherAssert.assertThat;
-
+/**
+ * @author Katarína Hermanová
+ * UČO 433511
+ * Github katHermanova
+ */
 @RunWith(SpringRunner.class)
 @DataJpaTest
 public class BookRepositoryTest {
 
-    @Autowired
-    private TestEntityManager entityManager;
-
     @Autowired
     private BookRepository bookRepository;
+    private Book book1;
+    private Book book2;
```

Comment:
do not use global variables

Reply:
but then I am going to have too much duplicated code or not? because I am using book1 in every test except the first one

Reply:
it is much more acceptable than this.

Reply:
It is much more acceptable than having it commot to all tests.

Reply:
I disagree but you are the boss so I will change it :)

Reply:
Thank you. Maybe I will change my mind if you can show me some source where it is done your way.

Reply:
I will try it with https://github.com/fi-muni/PA165/blob/master/eshop-service/src/test/java/cz/fi/muni/pa165/service/OrderServiceTest.java

Reply:
OK, I admit your that your way is also possible and valid. Now it is just a matter of preference.

Reply:
So is it ok if I change it back, please? :) please, please :)

Reply:
OK, as you wish